sdk/pineconevectordb/arm-pineconevectordb/review/arm-pineconevectordb-models.api.md

```diff
@@ -198,9 +218,9 @@ export interface UserAssignedIdentity {
 
 // @public
 export interface UserDetails {
-    emailAddress: string;
-    firstName: string;
-    lastName: string;
+    emailAddress?: string;
```

Comment:
@wanlwanl why there's no change log item for this change? Since this model is being used in both request and response, it should be treated as a breaking change, @kazrael2119 could you help create an issue for that? thanks

Reply:
https://github.com/Azure/azure-sdk-tools/issues/7883#issuecomment-2760334910